exchanges/order/order_types.go

```diff
@@ -191,6 +195,7 @@ type GetOrdersRequest struct {
 	Side       Side
 	StartTicks time.Time
 	EndTicks   time.Time
+	OrderId    string
```

Comment:
Can you please amend the comment to this struct for adding GetClosedOrders And add in AssetType asset.Item field mentioned above.

Reply:
Done